### PR TITLE
Docker stack 

### PR DESCRIPTION
**Pull request by ekkinox**
See https://oat-sa.atlassian.net/browse/NEX-995

Added:
- docker stack (nginx, fpm 7.4, redis and redis commander)
- README instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,61 @@
...
+After cloning this repository, you can build the [provided docker stack](docker-compose.yml):
 ```console
-$ composer install
+$ docker-compose up -d
 ```
+
+Then, install required dependencies with [composer](https://hub.docker.com/_/composer):
+```console
+$ docker run --rm --interactive --tty \
+  --volume $PWD:/app \
```
Comment (peetya) [inline README.md:35]: Just a side note, that on Windows it should be `--volume %cd%:/app`.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        command: ["redis-server", "--appendonly", "yes"]
+        hostname: demo_lti1p3_redis
+        networks:
+            - demo_lti1p3_network
+        ports:
+            - "6379:6379"
+        volumes:
+            - demo_lti1p3_redis_volume:/data
+
+    demo_lti1p3_redis_commander:
```
Comment (peetya) [inline docker-compose.yml:42]: Please put this container into `demo_lti1p3_network`, otherwise `redis-commander` won't see the `redis` container.

Comment (ekkinox) [reply docker-compose.yml:42]: it actually does, but yeah better be explicit

File: README.md (hunk 1 above)
Comment (ekkinox) [reply README.md:35]: will add it, the Windows population may be important customer side

**Pull request merged**